python/lsst/meas/base/pluginsBase.py

```diff
@@ -0,0 +1,100 @@
+import lsst.pex.config
+
+from .transforms import PassThroughTransform
+
+__all__ = ("BasePluginConfig", "BasePlugin")
+
+class BasePluginConfig(lsst.pex.config.Config):
+    """!
+    Base class measurement Plugin config classes.
+
+    Most derived classes will want to set defaults that make sense for the plugin type
+    """
+    pass
+
+class BasePlugin(object):
+    """!
+    Base class for measurement plugins.
+
+    This is the base class for SingleFramePlugin and ForcedPlugin; derived classes should inherit
+    from one of those.
+    """
+    # named class constants for execution order
+    CENTROID_ORDER = 0.0
+    SHAPE_ORDER = 1.0
+    FLUX_ORDER = 2.0
+    APCORR_ORDER = 3.0
+    CLASSIFY_ORDER = 4.0
+    DEFAULT_AFTERBURNER = 5.0
+
+    @classmethod
+    def getExecutionOrder(cls):
```

Comment:
Why a method instead of a variable?  Maybe just because that's the way we've always done it?  But I think a variable would be more pythonic (and it can always be methodised with `@property`).